apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -544,8 +554,12 @@ handle_block_response(Peer, _Peers, {ok, {{<<"200">>, _}, _, Body, _, _}}, block
 		Handled ->
 			Handled
 	end;
-handle_block_response(Peer, Peers, {ok, {{<<"200">>, _}, _, Body, _, _}}, {full_block, BI}) ->
-	case catch reconstruct_full_block(Peer, Peers, Body, BI) of
+handle_block_response(_, _, Response, _) ->
```

Comment:
Where is the new `handle_block_response/4` used?

Reply:
here https://github.com/ArweaveTeam/arweave/pull/213/files/4f1d52ab92356b126a45d0acfd229eb889532196#diff-bbfa4040e5f577553ef8de6f57e628a9R573-R574 - if response is not 200, ignore format and fall back on `handle_block_response/4`

Reply:
Ah, I see, thanks.